### PR TITLE
Color error messages

### PR DESCRIPTION
**Pull request by terror**

### CODE REVIEW

**Comment by casey** [comment]
@soenkehahn This copies the color from rust error messages. "error" in red and the ":" and message in bold.

File: tests/integration.rs [hunk 1]
```diff
@@ -1028,3 +1028,20 @@ fn listing_does_not_render_directory_file_sizes() {
...
   });
 }
+
+#[test]
+#[cfg(not(windows))]
+fn errors_printed_in_red_and_bold() {
+  use {executable_path::executable_path, std::process::Command};
+
+  let output = Command::new(executable_path("agora"))
+    .arg("--directory=/path/to/dir")
```
Comment (soenkehahn) [inline tests/integration.rs:1038]:
```suggestion
    .arg("--directory=/does/not/exist")
```

File: src/main.rs [hunk 2]
```diff
@@ -32,10 +35,18 @@ async fn main() {
...
+
+      error.print_backtrace(&mut stderr);
+
+      stderr
+        .set_color(ColorSpec::new().set_fg(Some(Color::Red)))
+        .ok();
+      write!(&mut stderr, "error").ok();
+
+      stderr.set_color(ColorSpec::new().set_bold(true)).ok();
+      write!(&mut stderr, ": {}", error).ok();
```
Comment (soenkehahn) [inline src/main.rs:48]: Currently there's no newline being printed after the error message, AFAICT.

**Review by soenkehahn — COMMENTED** [review]
Apart from my comments, LGTM.

**Pull request merged**